Gemfile

```diff
@@ -16,7 +17,7 @@ gem 'lograge'
 gem 'mitlibraries-theme'
 gem 'nokogiri'
 gem 'puma'
-gem 'rails', '5.1.7'
+gem 'rails', '5.2.4'
```

Comment:
I think there's a `5.2.4.2` out that patched a recent security vulnerability we should bump to while we're at it